apps/stats/fixtures/files/1093699/update_counts_by_app.hive

```diff
@@ -39,7 +39,7 @@
 2014-11-06	{fe9e9f88-42f0-40dc-970b-4b0e6b7a3d0b}	{ec8030f7-c20a-464f-9b0e-13a3a9e97384}	28.0.1	1	112
 2014-11-06	{fe9e9f88-42f0-40dc-970b-4b0e6b7a3d0b}	{ec8030f7-c20a-464f-9b0e-13a3a9e97384}	34.0	2	97
 2014-11-06	{fe9e9f88-42f0-40dc-970b-4b0e6b7a3d0b}	{3550f703-e582-4d05-9a08-453d09bdfdc6}	33.0	3	112
-2014-11-06	{fe9e9f88-42f0-40dc-970b-4b0e6b7a3d0b}	{92650c4d-4b8e-4d2a-b7eb-24ecf4f6b63a}	2.30	2	97
+2014-11-06	{fe9e9f88-42f0-40dc-970b-4b0e6b7a3d0b}	{92650c4d-4b8e-4d2a-b7eb-24ecf4f6b63a}	2.0	2	97
```

Comment:
We don't have the 2.30 version for seamonkey in our fixtures. Instead of updating the fixtures (used by many tests), i preferred updating this one.